### PR TITLE
Feat: Improve Vault card

### PR DESCRIPTION
**Pull request by FranRom**
- Add Vault Image example
- Add PancakeTag
- Add 6 decimals to Balle earned and LP staked
Screenshot 2021-04-26 at 23 34 42
Screenshot 2021-04-26 at 23 35 25

### CODE REVIEW

File: src/views/Farms/components/FarmCard/CardHeading.tsx [hunk 1]
```diff
@@ -21,20 +21,14 @@ const MultiplierTag = styled(Tag)`
...
-}) => {
+const CardHeading: React.FC<ExpandableSectionProps> = ({ lpLabel, multiplier, farmImage, tokenSymbol }) => {
   return (
     <Wrapper justifyContent="space-between" alignItems="center" mb="12px">
       <Image src={`/images/farms/${farmImage}.svg`} alt={tokenSymbol} width={64} height={64} />
       <Flex flexDirection="column" alignItems="flex-end">
         <Heading mb="4px">{lpLabel.split(' ')[0]}</Heading>
         <Flex justifyContent="center">
-          {isCommunityFarm ? <CommunityTag /> : <CoreTag />}
+          <PancakeTag />
```
Comment (jffernandez) [inline src/views/Farms/components/FarmCard/CardHeading.tsx:31]: This tag should be dynamic, will be a label on vault data with the name of the farm platform.

**Review by jffernandez — APPROVED** [review]
The vault tag should be dynamic. This maybe have to be reworked. Looks good for now.

File: src/views/Farms/components/FarmCard/CardHeading.tsx (hunk 1 above)
Comment (FranRom) [reply src/views/Farms/components/FarmCard/CardHeading.tsx:31]: do you mean in the FarmConfig? let me know the name and I'll add it. 
Something like
```
{
    farm-platform: 'pancake
}
```

Comment (jffernandez) [reply src/views/Farms/components/FarmCard/CardHeading.tsx:31]: Yes, it can be: 'farm'

Comment (FranRom) [reply src/views/Farms/components/FarmCard/CardHeading.tsx]: Refactor done!

**Review by cryptoballena — APPROVED** [review]

File: src/views/Farms/components/FarmCard/CardHeading.tsx (hunk 1 above)
Comment (jffernandez) [reply src/views/Farms/components/FarmCard/CardHeading.tsx]: Seems good!

**Pull request merged**